human_robot_collaboration/src/baxter_display.cpp

```diff
@@ -8,6 +8,7 @@
 #include <sensor_msgs/image_encodings.h>
 
 #include "human_robot_collaboration_msgs/ArmState.h"
+#include "ros_speech2text/status.h"
```

Comment:
where does this status thing comes from? Is it in speech to text? That's why travis is not compilng.